backend/debug.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# This script is used to perform remote debugging a Spring Boot application in debug mode in WSL or Linux environment
+mvn clean spring-boot:test-run -Dspring-boot.run.profiles=dev -Dspring-boot.run.jvmArguments="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```

Comment:
I would move this to the documentation section (as a helpful code snippet) not as a dedicated script at the root of the repo